pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -98,6 +100,10 @@ func (a *Operator) syncOperatorGroups(obj interface{}) error {
 	logger.WithField("targetNamespaces", targetNamespaces).Debug("updated target namespaces")
 
 	if namespacesChanged(targetNamespaces, op.Status.Namespaces) {
+		// Labels are only applied to namespaces that are not empty strings.
+		prunedStatusNamespaces := pruneEmptyStrings(op.Status.Namespaces)
```

Comment:
nit: since these strings are `namespaces`, `pruneEmptyNamespaces` is more contextual/ meaningful?